.github/ISSUE_TEMPLATE/release-plan.md

```diff
@@ -30,10 +32,21 @@ assignees: 'Guite, craigh'
 
 - [ ] Start release process in core manager at <https://ziku.la>
   - [ ] Create QA ticket
-  - [ ] Create release
-  - [ ] Copy assets (note: it now downloads assets from the chosen GitHub Actions build, since Jenkins has been removed)
+  - [ ] Create tag for core project
+  - [ ] Create tag for all slave repositories
```

Comment:
subtree split has a setting which is supposed to push tags to slaves. We were told to disable this when trying to troubleshoot the behaviors, but it didn't fix them. we could try re-enabling it.

Reply:
sounds better than inventing an additional mechanism, doesn't it?

Reply:
done